storage/client_split_test.go

```diff
@@ -186,7 +186,7 @@ func TestStoreRangeSplit(t *testing.T) {
 
 	// Split the range.
 	args, reply := adminSplitArgs(engine.KeyMin, splitKey, 1, store.StoreID())
-	if err := store.ExecuteCmd(args, reply); err != nil {
+	if err := store.ExecuteCmd(client.Call{Args: args, Reply: reply}); err != nil {
```

Comment:
In conjunction with your other proposed change, you might imagine that we change all of these tests to work directly with `client.GetCall` structs instead of the current helper methods (e.g. `getArgs`, `incrementArgs`, etc.)